tests/py/dynamo/conversion/test_index_select_aten.py

```diff
@@ -0,0 +1,38 @@
+import torch
+import torch.nn as nn
+from harness import DispatchTestCase
+from parameterized import param, parameterized
+from torch.testing._internal.common_utils import run_tests
+from torch_tensorrt import Input
+
+
+class TestIndexSelectConverter(DispatchTestCase):
+    @parameterized.expand(
+        [
+            ("1d_input", (10,), 0, (1,)),
+            ("2d_input_dim_0", (10, 3), 0, (0, 2)),
+            ("2d_input_dim_1", (5, 10), 1, (1, 2, 3)),
+            ("3d_input_dim_0", (10, 5, 10), 0, (0, 5)),
+            ("3d_input_dim_2", (10, 5, 10), 2, (3, 3, 4)),
```

Comment:
Add a test case for a negative `dim` input

Reply:
I have added a test case for a negative `dim` input and verified a test case. Thank you!